src/backend/distributed/planner/multi_planner.c

```diff
@@ -795,44 +829,20 @@ CreateAndPushPlannerRestrictionContext(void)
 
 
 /*
- * CurrentRelationRestrictionContext returns the the last restriction context from the
- * relationRestrictionContext list.
+ * CurrentRestrictionContext returns the the last PlannerRestrictionContext from the
+ * plannerRestrictionContextList list.
```

Comment:
The comments in this file are rather inconsistent but this is definitely not the last one in the list! Maybe "returns the most recently added PlannerRestrictionContext from the"